packages/gatsby-cli/src/index.ts

```diff
@@ -27,24 +27,29 @@ updateNotifier({ pkg }).notify({ isGlobal: true })
 const MIN_NODE_VERSION = `10.13.0`
 // const NEXT_MIN_NODE_VERSION = `10.13.0`
 
-if (!semver.satisfies(process.version, `>=${MIN_NODE_VERSION}`)) {
+const { version } = process
+
+if (
+  !semver.satisfies(version, `>=${MIN_NODE_VERSION}`, {
+    includePrerelease: true,
+  })
+) {
   report.panic(
     report.stripIndent(`
-      Gatsby requires Node.js ${MIN_NODE_VERSION} or higher (you have ${process.version}).
+      Gatsby requires Node.js ${MIN_NODE_VERSION} or higher (you have ${version}).
       Upgrade Node to the latest stable release: https://gatsby.dev/upgrading-node-js
     `)
   )
 }
 
-// if (!semver.satisfies(process.version, `>=${NEXT_MIN_NODE_VERSION}`)) {
-//   report.warn(
-//     report.stripIndent(`
-//       Node.js ${process.version} has reached End of Life status on 31 December, 2019.
-//       Gatsby will only actively support ${NEXT_MIN_NODE_VERSION} or higher and drop support for Node 8 soon.
-//       Please upgrade Node.js to a currently active LTS release: https://gatsby.dev/upgrading-node-js
-//     `)
-//   )
-// }
```

Comment:
This was commented out (and not removed) to avoid having to write new text copy for next time we deprecate node version - can we keep it commented out?